Vaibhav Shukla

```diff
@@ -0,0 +1 @@
+
```

Comment:
<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <title>Student Information Table</title>
  <style>
    table {
      width: 60%;
      border-collapse: collapse;
      margin: 20px auto;
    }
    th, td {
      border: 1px solid black;
      padding: 10px;
      text-align: center;
    }
    th {
      background-color: #f2f2f2;
    }
    tfoot {
      background-color: #eaeaea;
    }
  </style>
</head>
<body>

  <table>
    <thead>
      <tr>
        <th>Name</th>
        <th>Age</th>
        <th>Grade</th>
      </tr>
    </thead>
    <tbody>
      <tr>
        <td>Vaibhav</td>
        <td>16</td>
        <td>A</td>
      </tr>
      <tr>
        <td>Vibhu</td>
        <td>15</td>
        <td>B+</td>
      </tr>
      <tr>
        <td>Golu</td>
        <td>17</td>
        <td>A-</td>
      </tr>
    </tbody>
    <tfoot>
      <tr>
        <td colspan="3">End of Student Data</td>
      </tr>
    </tfoot>
  </table>

</body>
</html>